src/store/getters/state-getters.ts

```diff
@@ -237,3 +240,24 @@ export const getMaxStep = (state: any, getters: any): number => {
 export const getValidateSteps = (state: any, getters: any): boolean => {
   return state.stateModel.validateSteps
 }
+
+/** The current state of the certify section. */
+export const getCertifyState = (state: any): CertifyIF => {
+  return state.stateModel.certifyState
+}
+
+/** Business contact email address. */
+export const getBusinessEmail = (state: any, getters: any): string => {
+  return state.stateModel.defineCompanyStep.businessContact.email
```

Comment:
Is defineCompanyStep always declared? Is businessContact always declared? If not, use conditional chaining?

Reply:
Since we are dealing with a filing, they properties themselves are present but not the value. 
Either way though, optional chaining would not hurt! Added in next commit.

Reply:
Actually, I would rather not use optional chaining if it's not needed (since it can mask errors, such as unit test errors).

Reply:
Copy that!

Reply:
Removed 👍